pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/operator/OpChainTest.java

```diff
@@ -100,6 +100,20 @@ public void testStatsCollection() {
         Long.parseLong(executionStats.get(DataTable.MetadataKey.OPERATOR_EXECUTION_TIME_MS.getName())) <= 2000);
   }
 
+  @Test
+  public void testStatsCollectionTracingDisabled() {
+    OpChainExecutionContext context = OperatorTestUtil.getDefaultContextWithTracingDisabled();
+    DummyMultiStageOperator dummyMultiStageOperator = new DummyMultiStageOperator(context);
+
+    OpChain opChain = new OpChain(context, dummyMultiStageOperator, new ArrayList<>());
+    opChain.getStats().executing();
+    opChain.getRoot().nextBlock();
+    opChain.getStats().queued();
+
+    Assert.assertTrue(opChain.getStats().getExecutionTime() >= 1000);
+    Assert.assertEquals(opChain.getStats().getOperatorStatsMap().size(), 0);
+  }
```

Comment:
can you put more multistage operators in and mock the blocks?

Reply:
no i meant mock the last one but the rest are real. 
for example:
mailboxSend (mock exchange) --> project --> filter --> mailboxReceive (mock exchange) --> dummyBlockGeneratingOperator

Reply:
done